client/packages/editor/src/components/widgets/select/Select.css

```diff
@@ -0,0 +1,49 @@
+.select {
+  position: relative;
+  width: 100%;
+}
+.select-input {
+  display: flex;
+  width: 100%;
+}
+.select-button {
+  cursor: pointer;
+  padding: 10px;
+  font-size: 12px;
+  border-radius: var(--border-radius);
+  background: var(--input-bg);
+  display: inline-flex;
+  justify-content: space-between;
+  gap: 0.3rem;
+  width: 100%;
+}
+.select-button:focus {
+  box-shadow: var(--focus-shadow);
+}
+.select-menu {
+  overflow: auto;
+  width: 100%;
+  max-height: 20rem;
```

Comment:
I'm totally outdated in CSS ... never heard of 'rem' values

Reply:
root element font size. mostly in browsers 16px but can be changed by users. So if a user uses a bigger font-size also this rem value increases.

Reply:
thanks for the update :love_letter:  @ivy-lli